cluster/shard_mapper.go

```diff
@@ -230,10 +230,21 @@ func (r *RemoteMapper) NextChunk() (chunk interface{}, err error) {
 			}
 			aggValues = append(aggValues, v)
 		}
-		mo.Values = []*tsdb.MapperValue{&tsdb.MapperValue{
-			Value: aggValues,
-			Tags:  mvj[0].Tags,
-		}}
+
+		if mvj[0].Time != 0 {
```

Comment:
D'oh! I see it now, the old code was *not* setting the time field -- I never looked there. You are right. Thanks @li-ang

However, wouldn't this change work just as well:

```
mo.Values = []*tsdb.MapperValue{&tsdb.MapperValue{
+    Time: mvj[0].Time,
    Value: aggValues,
    Tags: mvj[0].Tags,
}}
```

In otherwords, just set the time.